src/main/java/org/apache/netbeans/nbm/repository/PopulateRepositoryMojo.java

```diff
@@ -389,16 +389,21 @@ else if ( skipLocalInstall )
                 String version = forcedVersion == null ? examinator.getSpecVersion() : forcedVersion;
                 String group = groupIdPrefix + ( examinator.isOsgiBundle() ? GROUP_EXTERNAL : examinator.hasPublicPackages() ? GROUP_API : GROUP_IMPL );
                 Artifact art = createArtifact( artifact, version, group );
+                ModuleWrapper wr = new ModuleWrapper( artifact, version, group, examinator, module, false );
                 if ( examinator.isOsgiBundle() )
                 {
                     Dependency dep = findExternal( module );
-                    if ( dep != null )
+                    if ( dep != null ) 
                     {
+                        
+                        art = createArtifact( artifact, dep.getVersion(), dep.getGroupId() );
+                        group = dep.getGroupId();
+                        version = dep.getVersion();
+                        wr = new ModuleWrapper( artifact, version, group, examinator, module , true );
                         // XXX use those coords instead of publishing this
                         // (for now all bundles are from Orbit, which does not publish to Central, or specially built)
```

Comment:
You an possibly remove the comment now as the job will now _"use those coords instead of publishing this"_

Reply:
done